cypress/steps/search-field-steps.ts

```diff
@@ -0,0 +1,60 @@
+export default class SearchFieldSteps {
+  static visit(useCustomTemplate: boolean = false) {
```

Comment:
I'd prefer to the tests and steps to be conditions free. Here you can just have two separate methods like `visit` and `visitWithTemplate` or something

Reply:
fixed